src/charm.py

```diff
@@ -69,6 +72,7 @@ def __init__(self, *args):
         self.framework.observe(self.on.config_changed, self._configure)
         self.framework.observe(self.on.gnbsim_pebble_ready, self._configure)
         self.framework.observe(self.on.start_simulation_action, self._on_start_simulation_action)
+        self.framework.observe(self._n2_requirer.on.n2_information_available, self._configure)
```

Comment:
We should probably also observe `fiveg_n2_relation_joined` events so that we adapt the status of the unit

Reply:
✅